src/x86/isa.c

```diff
@@ -46,6 +46,8 @@ struct cpuinfo_x86_isa cpuinfo_x86_detect_isa(
 		(max_base_index >= 7) ? cpuidex(7, 0) : (struct cpuid_regs){0, 0, 0, 0};
 	const struct cpuid_regs structured_feature_info1 =
 		(max_base_index >= 7) ? cpuidex(7, 1) : (struct cpuid_regs){0, 0, 0, 0};
+    const struct cpuid_regs structured_feature_info2 =
```

Comment:
indent is wrong.  should be 1 tab of 8 instead of 4 spaces.
copy the indent from the lines above